server/protocols/webadmin/webadmin-data/pom.xml

```diff
@@ -106,6 +106,11 @@
             <artifactId>equalsverifier</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
+            <artifactId>james-server-data-file</artifactId>
```

Comment:
we do you need data-file when you can use data-memory ?

Reply:
I'm using `sieveRepository = new SieveFileRepository(fileSystem);` in `SieveScriptRoutesTest`

Reply:
The real question is: why do you use a SieveFileRepository then